website/docs/add-secure-apps/providers/rac/rac_credentials_prompt.md

```diff
@@ -0,0 +1,93 @@
+---
+title: RAC Credentials Prompt
+---
+
+## About RAC Credentials Prompt
+
+The RAC provider can be configured to prompt users for their credentials when connecting to RAC endpoints. This is particulalry useful for establishing RDP connections to modern Windows systems that often require credentials to establish a connection.
```

Comment:
To avoid pasive voice we could say "You can configure the RAC provider to prompt..." but up to you....